docs/sources/configuration/integrations/gcp-exporter-config.md

```diff
@@ -0,0 +1,186 @@
+---
+aliases:
+- /docs/agent/latest/configuration/integrations/gcp-exporter-config/
+title: gcp_exporter_config
+---
+
+# gcp_exporter_config
+
+## Overview
+The `gcp_exporter_config` block configures the `gcp_exporter` integration, which is an embedded version of
+[`stackdriver_exporter`](https://github.com/prometheus-community/stackdriver_exporter). This allows for the collection of 
+metrics data from [GCP Cloud Monitoring (formerly stackdriver)](https://cloud.google.com/monitoring/docs).
+
+The exporter supports all metrics available via [GCP's monitoring API](https://cloud.google.com/monitoring/api/metrics_gcp). Metrics
+exporter by this integration will follow the template `stackdriver_<monitored_resource>_<metric_type_prefix>_<metric_type>`. 
+
+The following example shows a load balancing metric:
+
+![gcp-exporter-config-metric-example](https://grafana.com/media/docs/agent/gcp-exporter-config-metric-example.png)
+
+The following list shows its attributes: \
+monitored_resource = `https_lb_rule`\
+metric_type_prefix = `loadbalancing.googleapis.com/`\
+metric_type = `https/backend_latencies`
+
+These attributes result in a final metric name of:
+`stackdriver_https_lb_rule_loadbalancing_googleapis_com_https_backend_latencies`
+
+## Authentication
+
+The agent must be running in an environment with access to the GCP project it is scraping. The exporter
+uses the Google Golang Client Library, which offers a variety of ways to [provide credentials](https://developers.google.com/identity/protocols/application-default-credentials). Choose the option that works best for you.
+
+After deciding how the agent will obtain credentials, ensure the account is set up with the IAM role `roles/monitoring.viewer`.
+Since the exporter gathers all of its data from [GCP monitoring APIs](https://cloud.google.com/monitoring/api/v3), this is the only permission needed.
```

Comment:
```suggestion
After deciding how Agent will obtain credentials, ensure the account is set up with the IAM role `roles/monitoring.viewer`.
Since the exporter gathers all of its data from [GCP monitoring APIs](https://cloud.google.com/monitoring/api/v3), this is the only permission needed.
```